exchange/bitswap/bitswap.go

```diff
@@ -208,6 +208,12 @@ func (bs *Bitswap) GetBlocks(ctx context.Context, keys []key.Key) (<-chan *block
 		log.Event(ctx, "Bitswap.GetBlockRequest.Start", &k)
 	}
 
+	go func() {
```

Comment:
Actually, the timeout is meant for the commands in general, not specific to bitswap.